PWG/EMCAL/EMCALtasks/AliEmcalAodTrackFilterTask.cxx

```diff
@@ -137,15 +121,20 @@ AliEmcalAodTrackFilterTask* AliEmcalAodTrackFilterTask::AddTaskEmcalAodTrackFilt
     aodTask->SetTracksOutName(name);
     aodTask->SetTracksInName(inname);
     aodTask->SetMC(kFALSE);
-  
+    TString trackname(inname);
+   if(!trackname.IsNull())aodTask->AddParticleContainer(trackname);
+
+    TString OutTrackName(name);
+   if(!OutTrackName.IsNull())aodTask->AddParticleContainer(OutTrackName);
+
     Bool_t includeNoITS  = kFALSE;
     Bool_t doProp        = kFALSE; //force propagation of all tracks to EMCal
     Bool_t doAttemptProp = kTRUE;  //only propagate the tracks which were not propagated during AOD filtering
     Bool_t isMC          = kFALSE;
   
     TString runPeriod(runperiod);
     runPeriod.ToLower();
-  if (runPeriod == "lhc10b" || runPeriod == "lhc10c" || runPeriod == "lhc10d" ||
```

Comment:
Unless required to make it work for run 2, I wouldn't recommend commenting out Run 1 code. This isn't especially robust code, but there are still folks doing some run 1 work, so I wouldn't recommend throwing out code that still works, even if not relevant for you

Reply:
As far as I can tell this code does not interfere with running over run 2 so I will reinstate it. Thank you for the comment